bakery_lint/fonttests/test_ttf.py

```diff
@@ -68,11 +68,11 @@ class TTFTestCase(TestCase):
 
     @tags('info',)
     def test_version_in_name_table_is_in_correct_format(self):
-        """ Version is in correct format in `name` table """
+        """ Version format is correct in NAME table? """
         ttfont = ttLib.TTFont(self.operator.path)
 
         def is_valid(value):
-            return re.match(r'Version\s[1-9]+\.\d+', value)
+            return re.match(r'Version\s0*[1-9]+\.\d+', value)
```

Comment:
This seems wrong. A version of  `0.001` is invalid, the smallest valid value is `1.000`